versions.txt

```diff
@@ -4,6 +4,9 @@ cloudwatch-agent=1.300041.0b681
 # Represents the current release of the CloudWatch Agent Operator.
 operator=1.4.1
 
+# Represents the current release of the Target Allocator.
+targetallocator=0.90.0
```

Comment:
I initially set it as `1.0.0`, but because we're currently using upstream's image, we have to use this version. I can add a comment to note that this should be changed once we use our own image for the `targetallocator`.